test/smoke/packages/contour/1.18.2/contour-test.sh

```diff
@@ -42,18 +43,19 @@ spec:
 EOF
 
 sleep 15s
-kubectl --namespace projectcontour port-forward svc/envoy 5436:80 &
+kubectl --namespace projectcontour port-forward svc/envoy 5436:80 & pid=$1
```

Comment:
I think the `pid` has to be assigned like this

```suggestion
kubectl --namespace projectcontour port-forward svc/envoy 5436:80 & pid=$!
```

Also, we can use a new line I think, with a specific name for `pid`

```suggestion
kubectl --namespace projectcontour port-forward svc/envoy 5436:80 &
portforward_pid=$!
```

or similar